### PR TITLE
Strip elements not supported by Sablon from trix-input.

### LINKED ISSUE #1628: Server-Side Validation für Trix-Input
Bisher beschränkt sich hier die Server-Seite auf Sicherheitsmassnahmen (XSS, ...) . Als nächste Ausbaustufe wäre es sinnvoll von Sablon nicht unterstützte Tags auch nicht zuzulassen. *(Sablon is im Moment absichtlich so gebaut, dass es bei nicht unterstützem Markup Fehler wirft.)*

Im Moment sind die folgenden Tags unterstützt:
- b
- br
- div
- em
- i
- li
- ol
- p
- strong
- u
- ul

### PR DESCRIPTION
**Pull request by deiferni**
This PR strips elements that are not supported by Sablon from trix-input.

Also log to sentry when this happens. We don't expect markup to be stripped since the markup generated by trix should always be supported by Sablon.

Currently supported tags are:
- b
- br
- div
- em
- i
- li
- ol
- p
- strong
- u
- ul

Closes #1628.

### CODE REVIEW

**Comment by deiferni** [comment]
@phgross or @lukasgraf plz have a look _(i'll rebase later)_

File: opengever/base/transforms/trix2sablon.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+from opengever.base.sentry import log_msg_to_sentry
+from Products.PortalTransforms.data import datastream
+from Products.PortalTransforms.transforms.safe_html import SafeHTML
+from zope.globalrequest import getRequest
+
+
+# These are the HTML tags that are supported by sablon
```
Comment (lukasgraf) [inline opengever/base/transforms/trix2sablon.py]: Maybe add the meaning of the values as a comment? From `SafeHTML`:

> `Value is 1 if they have a closing part (e.g. <p>...</p>) and 0 for empty tags (like <br />).`

File: opengever/base/transforms/trix2sablon.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+def convert(markup):
+    """Use this function to transform markup from trix to markup that can
+    be processed by sablon.
+
+    This converter is expected to do nothing since trix markup is already valid
+    for sablon. It is just a safeguard against malicious markup injection or
+    against changes in trix.
+
+    Thus we also log to sentry whenever we acutally have to convert markup.
```
Comment (lukasgraf) [inline opengever/base/transforms/trix2sablon.py]: `acutally` -> `actually`

**Comment by lukasgraf** [comment]
@deif some minor points, but other than that: Looks good to me! 👍

**Comment by lukasgraf** [comment]
Example sentry event also looks good, I removed it.

**Comment by deiferni** [comment]
@lukasgraf updated

**Pull request merged**